### PR TITLE
API Update API to reflect changes to CLI interaction

### PR DESCRIPTION
**Pull request by GuySartorelli**
Reflects changes in https://github.com/silverstripe/silverstripe-framework/pull/11353

## Issue
- https://github.com/silverstripe/silverstripe-framework/issues/11341

### CODE REVIEW

File: docs/en/07_tips_and_tricks.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ This obfuscation is handled by the [`NameObfuscator`](api:SilverStripe\GraphQL\S
 There are various implementations:
 
 - [`NaiveNameObfuscator`](api:SilverStripe\GraphQL\Schema\Storage\NaiveNameObfuscator): Filename/Classname === Type name (debug only)
-- [`HybridNameObfuscator`](api:SilverStripe\GraphQL\Schema\Storage\HybridNameObfuscator): Filename/Classname is a mix of the typename and a md5 hash (default).
+- [`PolyNameObfuscator`](api:SilverStripe\GraphQL\Schema\Storage\PolyNameObfuscator): Filename/Classname is a mix of the typename and a md5 hash (default).
```
Comment (emteknetnz) [inline docs/en/07_tips_and_tricks.md]: This wasn't nor should be renamed

File: src/Schema/Storage/HybridObfuscator.php [hunk 2]
```diff
@@ -6,7 +6,7 @@
 /**
  * Hashed for less discoverability, but still readable if you focus on it
  */
-class HybridObfuscator implements NameObfuscator
+class PolyObfuscator implements NameObfuscator
```
Comment (emteknetnz) [inline src/Schema/Storage/HybridObfuscator.php]: Revert naming change

**Review by emteknetnz — CHANGES_REQUESTED** [review]

File: docs/en/07_tips_and_tricks.md (hunk 1 above)
Comment (ChloeSartorelli) [reply docs/en/07_tips_and_tricks.md]: Oops yes, good catch.

Comment (ChloeSartorelli) [reply docs/en/07_tips_and_tricks.md]: Fixed

File: src/Schema/Storage/HybridObfuscator.php (hunk 2 above)
Comment (ChloeSartorelli) [reply src/Schema/Storage/HybridObfuscator.php]: Fixed

**Review by emteknetnz — APPROVED** [review]

**Pull request merged**